### ISSUE TITLE
Add option for linking more elements in Icon/Image, text and link

### ISSUE THREAD

**Issue by mabolek** [t3kit/theme_t3kit#47]
It would be good for the user experience to link more elements in the _Icon, Text and Button_, _Image text and link_ and _Icon, Text and Link_ content elements.

It seems that by convention, users expect these kind of graphic+text+link to be clickable. This is especially true for the title and image/icon.

Making the entire area clickable reflects convention, e.g. on news sites (e.g. bbc.com) or navigation matrices (e.g. danskebank.dk)

**Comment by dmh** [comment]
Hi,

I think it is good idea, and we should implement such links into these elements. For now the question is how to implement it right and without breaking changes compare to previous versions of those elements.

As for me I don't like how it is implamented on this site `danskebank.dk`. Because they are using `js` and dynamicly adding links to `div` tag. Seems it is wrong way. Second variant with **BBC site** is more interesting, but for us it could be also tricky, because with this **BBC** approach we can have two same links in one element, but might be it is depend on content, so it is not a big issue.

So, to summarize, yes we will try to implement such **"wrapper links"**.

Thanks!

**Comment by mabolek** [comment]
Hi!

I agree with your thoughts about the technical implementations at danskebank.dk and BBC. But: The user experience is good.

Regarding implementation, it's one of these cases where you can get a lot of `<f:if><f:link.action></f:if>` wraps messing up the tag nesting. I wonder if it wouldn't be good to introduce a "disabled" argument to the fluid link view helper. Something like:

`<f:link.action action="foo" disabled="{settings.linkOrNot}">Link</f:link.action>`

A `disabled="true"` would just pass the contents (i.e. the "Link" text) 
without adding the `<a>` tag.

**Comment by yar-trach** [comment]
Hi! I've added this feature to theme + felayout_t3kit and felayout_bluemountain
